lib/file-info-view.coffee

```diff
@@ -9,35 +9,43 @@ class FileInfoView extends HTMLElement
     @currentPath.classList.add('current-path')
     @appendChild(@currentPath)
 
-    @handleCopiedTooltip()
-
     @activeItemSubscription = atom.workspace.onDidChangeActivePaneItem =>
       @subscribeToActiveItem()
     @subscribeToActiveItem()
 
-    clickHandler = =>
-      text = @getActiveItemCopyText()
+    clickHandler = (event) =>
+      altClick = if process.platform is 'darwin' then event.metaKey else event.ctrlKey
```

Comment:
I think it's fine the way it is right now - `cmd` is equivalent to `ctrl` on Windows/Linux.